app/views/chapters/content.json.jbuilder

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+json.chapter do
+  # :chapter => {:id, :number, :title, :en_title, :en_short_summary, :en_short_summary, :artwork_url}
+  json.call(@chapter, :id, :number, :title, :en_title, :en_short_summary, :en_short_summary, :artwork_url)
+
+  # :book => {:id, :sequence, :title, :en_title, :en_short_summary, :en_synopsis, :artwork_url}
+  json.book(@chapter.book, :id, :sequence, :title, :en_title, :en_short_summary, :en_synopsis, :artwork_url)
+
+  # :chhands => {:id, :sequence, :vaak}
+  json.chhands @chapter.chhands.order(:sequence => :ASC) do |chhand|
+    json.call(chhand, :id, :sequence, :vaak)
+    json.name(chhand.chhand_type.name)
+
+    # :pauris => {:id, :number, :translation}
+    json.pauris chhand.pauris.order(:sequence => :ASC) do |pauri|
+      json.call(pauri, :id, :number, :translation)
+      json.footnote do
+        json.call(pauri.external_pauri, :bhai_vir_singh_footnote, :content, :original_content)
```

Comment:
So, I have a few comments: 

### External Pauri Shouldn't Be in the `footnotes` Object

So, the `external_pauri` is a rare case when Kavi Ji added verses from other granths directly SPG. 

See more here: https://github.com/vidhiya-saagar/spg2/issues/42

From the way it looks, it's going to look like:
```js
...{
      id: 1, 
      number: 1, 
      translation: {},
     footnote: {
       external_pauri: {}...
     }
     tuks: {}
   }
```

Reply:
I don't think `pauri` has `footnotes` from the schema, that's why I did this, let me just double check

Reply:
![image](https://user-images.githubusercontent.com/66918439/236694246-e6d3314c-4ef0-4323-8f02-74bedac6b43a.png)

Reply:
Ohhh you are right! That `pauri_footnotes` stuff never got merged :P

Reply:
How should we resolve this then?

Reply:
Since we just meged #52, are you able to ro do `pauri.footnote` here?

Reply:
Didn't get a chance to test it out yet, I'll do it in about 5ish hours